test/parallel/test-repl-.save.load.js

```diff
@@ -74,5 +74,20 @@ putIn.write = function(data) {
 };
 putIn.run(['.load ' + loadFile]);
 
+// clear the REPL
+putIn.run(['.clear']);
+
+// NUL (\0) is disallowed in filenames in UNIX-like operating systems and 
+// Windows so we can use that to test failed saves
+var invalidFileName = join(common.tmpDir, '\0\0\0\0\0');
```

Comment:
Sure, as long as that won't get flagged as non-idiomatic for the file. Other similar values are declared with `var` and I don't want to refactor them for an otherwise unrelated change. Change set focused on one and only one issue and all that...